game/scripts/npc/abilities/phantom_assassin_coup_de_grace.txt

```diff
@@ -19,7 +19,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "crit_chance"                                     "15"
+        "crit_chance"                                     "10"  //OAA
```

Comment:
Remove the 25 talent, leave this crit chance unchanged.

Reply:
And replace it with what? With this change you at least gain 15% chance back when you pick up the talent.

Reply:
Anything at all, True Strike, All stats, Agility, attack speed, Linkens Sphere, Armour, CDR. 10% Chance will ruin the hero pre 25.

Reply:
RNG is the nature of the hero. Crit or not crit. Phantom Strike nerf hurt her more. Do I need to explain again why PA is the strongest hero in the game? Even if remove her ult she would still be picked.